core/src/test/java/org/pentaho/platform/util/EmailerTest.java

```diff
@@ -22,13 +22,39 @@
 
 import org.junit.Test;
 
+import javax.mail.MessagingException;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.UUID;
+
```

Comment:
I'd copy the setup function that scheduler/.../email/EmailerTest.java had

`  @Before
  public void setUp() {
    emailer = new Emailer();


    MockMail.clear();
  }`

That way your test is isolated from changes in other test cases. So a test case will pass or fail independently of whether you run all tests or a subset or particular order.

https://github.com/pentaho/pentaho-platform/blob/master/scheduler/src/test/java/org/pentaho/platform/scheduler2/email/EmailerTest.java#L63-L68